src/screens/tasks.tsx

```diff
@@ -1,8 +1,105 @@
 import React from 'react'
-import { Switch, Route, useRouteMatch } from 'react-router-dom'
+import { Switch, Route, useRouteMatch, Link } from 'react-router-dom'
 // components
 import Layout from 'components/layout'
-import Task from 'domain/tasks/task'
+import TaskD from 'domain/tasks/task'
+//types
+import { Task } from 'types/tasks'
```

Comment:
Replace code in line seven by ```import { Task as TaskModel } from 'types/tasks' ``` this way we avoid having to call Task component TaskD